src/main.rs

```diff
@@ -23,7 +23,6 @@ fn main() {
             for c in time.chars() {
                 let col = match c {
                     '0'..='9' => c as usize - '0' as usize,
-                    ':' => 10,
```

Comment:
Indeed, it is not necessary to have the colon in this case as a separate entry, but it does improve the understanding of the code.

The optimiser will produce the same code regardless.